app/Models/Server.php

```diff
@@ -94,6 +94,14 @@ protected static function booted()
                     ]);
                 }
             }
+            if (!isset($server->proxy->redirect_enabled)) {
+                $server->proxy->redirect_enabled = true;
+            }
+        });
+        static::retrieved(function ($server) {
+            if (!isset($server->proxy->redirect_enabled)) {
+                $server->proxy->redirect_enabled = true;
+            }
```

Comment:
Is there a better way to make sure this is true by default?

Reply:
Yeah, this is fine.